### ISSUE TITLE
TextField prop helperText used only for error

### ISSUE THREAD

**Issue by agraver** [lookfirst/mui-rff#22]
**Describe the bug**
https://github.com/lookfirst/mui-rff/blob/6adb9c6bfdef0c4fb83d29e5f200452dfa9149fa/src/TextField.tsx#L30

```
<TextField
    ...
    helperText="Help, not an error"
/>
```
should work, i.e.

```
      <TextField
        id="standard-full-width"
        label="Label"
        style={{ margin: 8 }}
        placeholder="Placeholder"
        helperText="Full width!"
        fullWidth
        margin="normal"
        InputLabelProps={{
          shrink: true,
        }}
      />
```

**Comment by lookfirst** [comment]
Maybe I'm a bit confused, but this one does seem to work as you can see that `{...rest}` is included in there. So if you override helperText, it should get set.

![image](https://user-images.githubusercontent.com/85355/68731394-7b816400-0602-11ea-9c7d-0d7fae9bf0bb.png)

**Comment by lookfirst** [comment]
Ok, I think this has been fixed in v1.4.2. Let me know if I missed anything and we can reopen it easily.
